js/clipboard.js

```diff
@@ -10,6 +10,15 @@
       this.$menuClearAll = this.$element.find('li#node_clearClipboard a');
       this.$toggleButtons = $('button.clipboard');
 
+      this.storage = localStorage;
```

Comment:
Wow, this is all it takes to use local storage, eh? Very cool. 😮

Reply:
Yes, that's it :)

And we could replace it in this line by `sessionStorage` if we prefer that one.